content/docs/join_us/recruitment_FAQs.md

```diff
@@ -134,32 +234,121 @@ The team's role at the Turing is to collaborate with institute researchers to su
 It is much, much more usual for team members to be working on other people's projects than their own.
 If you would like to prioritise pursuing your own research agenda at the Turing there are [other types of positions](https://www.turing.ac.uk/opportunities-turing) at the Institute that allow you to do this.
 
+<!-- TODO: This probably needs an update from Martin or somebody senior; I'm not sure if this is really true in T2.0 -->
```

Comment:
Yes, we should delete the paragraph in lines 238-244. In T2.0 we expect our work to be fully aligned to the grand challenges and thus we should be clear about this expectation when joining the team. I think you address this in the right way by encouraging those interested in research roles to apply for specific ones relevant to them in line 235.